tensorflow/python/tpu/bfloat16.py

```diff
@@ -70,11 +70,11 @@ def inner_custom_getter(getter, *args, **kwargs):
 
 @tf_export(v1=['tpu.bfloat16_scope'])
 @tf_contextlib.contextmanager
-def bfloat16_scope():
+def bfloat16_scope(name=''):
```

Comment:
Let's make the default None here, and then set to `''` if None, as per the API policy: https://github.com/tensorflow/community/blob/master/governance/api-reviews.md#optional-arguments-with-default-values

Reply:
I used '' instead of "None" is because of backward compatibility. Since there are already some user use the tensorflow without my change to train their bfloat16 model, they didn't provide the bfloat16 namescope name, therefore in their trained model, the name of bfloat16 namescope is default value ''. If I change the '' into "None", they have to provide a name to their bfloat16 namescope, and that will cause the node name change in their models, and the inference will fail.  If they don't provide the name, Tensorflow will raise error like follows:
tensorflow/python/ops/variable_scope.py", line 2278, in __init__
    raise TypeError("If default_name is None then name_or_scope is required")